src/util/WebSocket.ts

```diff
@@ -0,0 +1,20 @@
+import { formatLeaderboardTime } from './TimeUtil';
+
+export function mapPhaseEnumToString(phase: number) : string {
+    const phases = ["NotRunning", "Running", "Ended", "Paused"];
+    if (phase > 0 && phase < phases.length) {
+        return phases[phase];
+    } else {
+        return "";
+    }
+}
```

Comment:
Couldn't we just do:
```
return phases[phase] || "";
```

Reply:
Ideally we should use `expect(phases[phase], "Unknown timer phase supplied")` here or so, as the index out of bounds case should visibly explode and not implicitly do something weird.